src/history.jl

```diff
@@ -256,8 +256,8 @@ nrests(ch::RestartedHistory) = Int(ceil(ch.iters/ch.restart))
 Determine whether a collection `x` is plotable. Only vectors and matrices are
 such objects.
 """
-plotable{T<:Real}(::VecOrMat{T})::Bool = true
```

Comment:
To be honest I didn't know how to combine `where` syntax with return type annotations. But I don't think we need explicit return type annotations for such a simple function anyways.